wondrous-app/components/Common/CornerWidget/Component.tsx

```diff
@@ -0,0 +1,122 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import CloseModalIcon from 'components/Icons/closeModal';
+import get from 'lodash/get';
+import { useRouter } from 'next/router';
+
+import palette from 'theme/palette';
+import typography from 'theme/typography';
+import { ENTITIES_TYPES } from 'utils/constants';
+
+import { CornerWidgetComponentType } from './types';
+
+const addedToText = (type) => {
```

Comment:
this function is unnecessary, it's only called one, easier to just write out the logic in line